jupyter/car-iar-poisson/car-iar-poisson.ipynb

```diff
@@ -0,0 +1,1310 @@
+    "with open(\"data/scotland_data.json\") as file:\n",
+    "    scotland_data = json.load(file)\n",
+    "\n",
+    "def mungeCARdata4stan(adjBUGS, numBUGS):\n",
+    "    N = len(numBUGS)\n",
+    "    nn = numBUGS\n",
+    "    N_edges = len(adjBUGS) // 2\n",
+    "    node1 = np.zeros(N_edges, dtype=int)\n",
+    "    node2 = np.zeros(N_edges, dtype=int)\n",
+    "    iAdj = 0\n",
+    "    iEdge = 0\n",
+    "    \n",
+    "    for i in range(N):\n",
+    "        for j in range(nn[i]):\n",
+    "            iAdj += 1\n",
+    "            if i + 1 < adjBUGS[iAdj - 1]:\n",
+    "                iEdge += 1\n",
+    "                node1[iEdge - 1] = i + 1\n",
+    "                node2[iEdge - 1] = adjBUGS[iAdj - 1]\n",
+    "    \n",
+    "    return {\n",
+    "        \"N\": N,\n",
+    "        \"N_edges\": N_edges,\n",
+    "        \"node1\": node1,\n",
+    "        \"node2\": node2\n",
+    "    }"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "13:50:24 - cmdstanpy - INFO - CmdStan start processing\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "ef77d970867840eaaf354be8c8e88103",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "chain 1 |          | 00:00 Status"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "960808df4af94c50866a686f4d72b651",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "chain 2 |          | 00:00 Status"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "e9de4a91783c413f80930b7a2c88ddba",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "chain 3 |          | 00:00 Status"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "528431e37032475289fa4ef64bb15620",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "chain 4 |          | 00:00 Status"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "                                                                                                                                                                                                                                                                                                                                "
```

Comment:
before checking in a jupyter notebook, please reload and delete all outputs.  this makes it much easier to maintain and for others to use.

Reply:
Noted, I already removed the outputs.